### ISSUE TITLE
Migrate to TYPO3 9.x and PHP 7.2

### ISSUE THREAD

**Issue by nicozerr** [cobwebch/flush_language_cache#6]
Hi.

TYPO3 9.x has been released. Will ext:flush_language_cache be migrated for this version ?

Thank you.

**Comment by fsuter** [comment]
Hi Nicolas. It definitely will. I generally wait for the LTS version to be released before I do upgrade work, because I have little time available. Thus I try to avoid doing double work because something was changed in some other milestone release along the way.
But maybe you are willing to test it and report about compatibility?

**Comment by stoppeye** [comment]
I just tried it with a 9.5 LTS. I installed the latest version of your extension (ignoring the warnings...).
So far, I only tested the button in the BE. It gets displayed correctly. I tested it by changing a label in one of my extensions. After clicking the BE button, an error flash message appeared ("An error occurred. An error occurred while clearing the cache. It is likely not all caches were cleared as expected."). The message in the syslog was:

`Core: Exception handler (WEB): Uncaught TYPO3 Exception: Return value of TYPO3\CMS\Backend\Http\RouteDispatcher::dispatch() must implement interface Psr\Http\Message\ResponseInterface, null returned | TypeError thrown in file C:\T3\core\9LTS\typo3_src-9.5.1\typo3\sysext\backend\Classes\Http\RouteDispatcher.php in line 87. Requested URL: https://t3dev-v9.eye.ch/typo3/index.php?route=%%2Fajax%%2Fflush_language_cache%%2Fclear&token=--AnonymizedToken--` 

However, the cache was cleared successfully. The label was updated as expected.
